server/pbenchinacan/load_keycloak.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env -S bash -e
+
+# This scripts loads the keycloak server with following configuration
+# 1. A realm with the name pbench
+# 2. A client pbench-client under the realm pbench
+# 3. A user test_admin
```

Comment:
Once again, I don't like this `test_admin` name; we're going to need to expose admin access for the pbench realm because it's needed for auditing and config settings. While we could create a separate admin user for "testing" I don't see the point, and we definitely need a "real" admin.